tuner/tuner/candidate_gen_test.py

```diff
@@ -27,9 +27,12 @@ def tuner_ctx() -> Generator[common.TunerContext, None, None]:
     from logging import Logger
     from unittest.mock import MagicMock
 
-    with ir.Context() as ctx:
-        logger: Logger = MagicMock(spec=Logger)
-        yield common.TunerContext(ctx, logger)
+    # Mock the logger
+    mock_logger = MagicMock(spec=Logger)
+
+    # Use TunerContext with the mocked logger
```

Comment:
I don't think these comments help here

```suggestion
    mock_logger = MagicMock(spec=Logger)
```